exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -2,190 +2,194 @@
+                    Console.WriteLine($"{players[currentPlayer]} is getting out of the penalty box");
 
-                    Console.WriteLine(players[currentPlayer]
-                            + "'s new location is "
-                            + places[currentPlayer]);
-                    Console.WriteLine("The category is " + CurrentCategory());
+                    AdvanceOnGameBoard(roll);
+                    IsBackToStartOfBoard();
+
+                    Console.WriteLine($"{players[currentPlayer]}'s new location is {places[currentPlayer]}");
+                    Console.WriteLine($"The category is {Category()}");
                     AskQuestion();
                 }
                 else
                 {
-                    Console.WriteLine(players[currentPlayer] + " is not getting out of the penalty box");
+                    Console.WriteLine($"{players[currentPlayer]} is not getting out of the penalty box");
                     isGettingOutOfPenaltyBox = false;
                 }
-
             }
             else
             {
+                AdvanceOnGameBoard(roll);
+                IsBackToStartOfBoard();
 
-                places[currentPlayer] = places[currentPlayer] + roll;
-                if (places[currentPlayer] > 11) places[currentPlayer] = places[currentPlayer] - 12;
-
-                Console.WriteLine(players[currentPlayer]
-                        + "'s new location is "
-                        + places[currentPlayer]);
-                Console.WriteLine("The category is " + CurrentCategory());
+                Console.WriteLine($"{players[currentPlayer]}'s new location is {places[currentPlayer]}");
+                Console.WriteLine($"The category is {Category()}");
                 AskQuestion();
             }
 
         }
 
         private void AskQuestion()
         {
-            if (CurrentCategory() == "Pop")
+            if (Category() == "Pop")
             {
                 Console.WriteLine(popQuestions.First());
                 popQuestions.RemoveFirst();
             }
-            if (CurrentCategory() == "Science")
+            if (Category() == "Science")
             {
                 Console.WriteLine(scienceQuestions.First());
                 scienceQuestions.RemoveFirst();
             }
-            if (CurrentCategory() == "Sports")
+            if (Category() == "Sports")
             {
                 Console.WriteLine(sportsQuestions.First());
                 sportsQuestions.RemoveFirst();
             }
-            if (CurrentCategory() == "Rock")
+            if (Category() == "Rock")
             {
                 Console.WriteLine(rockQuestions.First());
                 rockQuestions.RemoveFirst();
             }
         }
 
 
-        private string CurrentCategory()
+        private string Category()
+        {
+            if (places[currentPlayer] == 0 || places[currentPlayer] == 4 || places[currentPlayer] == 8) return "Pop";
```

Comment:
Maybe places[currentPlayer] could be assign to a local variable, Mark did say that is a trade-off because it also add a line or two.

Reply:
good idea